RSS/Atom error in /firefox/nightly/notes/feed/

## Description

https://github.com/mozmeao/www-site-checker/actions/runs/6648592312

```
Checking https://***/firefox/nightly/notes/feed/
Invalid feed detected:
https://***/firefox/nightly/notes/feed/ mismatched tag @ L:27
Error: Process completed with exit code 128.
```

Looks like it might be due to an `<img>` element which isn't closed correctly for XHTML

```
<img alt="UI setting for GPC" src="https://www.mozilla.org/media/img/firefox/releasenotes/note-images/120_relnote_GCP.png">
```

--- a/bedrock/firefox/templates/firefox/releases/nightly-feed.xml
+++ b/bedrock/firefox/templates/firefox/releases/nightly-feed.xml
@@ -24,12 +24,12 @@
     <id>{{ note.link|absolute_url }}</id>
     <link rel="alternate" type="text/html" href="{{ note.link|absolute_url }}"/>
     <updated>{{ note.modified.isoformat() }}</updated>
-    <content type="xhtml">
-      <div xmlns="http://www.w3.org/1999/xhtml">
+    <content type="html">
+      <![CDATA[
         {{ note.version }} / {{ note.tag|replace('HTML5', 'Web Platform') }}
         {%- if note.bug %} / <a href="https://bugzilla.mozilla.org/show_bug.cgi?id={{ note.bug }}">Bug {{ note.bug }}</a>{% endif %}
         {{ note.note|markdown|safe }}
-      </div>
+      ]]>
     </content>
   </entry>
   {% endfor %}

